Src/IronPython/Runtime/Types/DocBuilder.cs

```diff
@@ -402,6 +402,10 @@ internal static class DocBuilder {
                 type = type.GetElementType();
             }
 
+            if (type.IsGenericParameter) {
```

Comment:
Do we still need this check? I guess it's fine since prevents the PythonType from being created.

Reply:
Yes, it is a fast track now, still useful. Generic parameter types have trivial names.